runners/direct-java/src/test/java/org/apache/beam/runners/direct/WindowEvaluatorFactoryTest.java

```diff
@@ -304,6 +305,12 @@ public boolean isCompatible(WindowFn<?, ?> other) {
     }
 
     @Override
+    public void verifyCompatibility(WindowFn<?, ?> other) throws IncompatibleWindowException {
+      throw new IncompatibleWindowException(
+          other, "EvaluatorTestWindowFn is not compatible with any other WindowFn.");
```

Comment:
Here, too, use `String.format` and use the name of the class instead of typing it out. Same with `WindowFn`.